src/styles/blocks/stars.css

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background: url(../../images/star-active.svg) no-repeat center;
+}
+
+.stars__star:last-child {
+  background-image: url(../../images/star.svg);
+  margin-right: 0;
+}
```

Comment:
use the styles from the 'stars' task, to cover cases of different numbers of stars

Reply:
> use the styles from the 'stars' task, to cover cases of different numbers of stars

i don't understand what you mean...

Reply:
And why I didn't pass the tests after pull requests ???
Because my test report is without fail.

Reply:
<img width="899" alt="image" src="https://user-images.githubusercontent.com/70220667/237019945-572adcf4-d87e-4d2e-b70c-a15d223921d8.png">

Reply:
> <img alt="image" width="899" src="https://user-images.githubusercontent.com/70220667/237019945-572adcf4-d87e-4d2e-b70c-a15d223921d8.png">

I didn't add a modifier for the map block "stars" because it's not needed here (if I understood it well). But I added modifiers for each star block "stars"